docs/content/platform/blocks/github/checks.md

```
@@ -0,0 +1,75 @@
+# Github Create Check Run
+
+### What it is
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)